addons/medical_feedback/functions/fnc_playInjuredSound.sqf

```diff
@@ -28,6 +28,15 @@ if (!local _unit) exitWith {
 };
 if !(_unit call EFUNC(common,isAwake)) exitWith {};
 
+// Limit network traffic by only sending the event to players who can potentially hear it
+private _distance = if (_type == "hit") then {
+    [50, 60, 70] select _severity;
+} else {
+    [10, 15, 20] select _severity;
+};
+private _targets = _unit nearEntities ["CAManBase", _distance];
+if (_targets isEqualTo []) exitWith {};
```

Comment:
There always is at least one unit near, the unit that is hurt. Also, the avatar should still hear himself moan, even if there is no one else nearby.

Reply:
`nearEntities` also completely fails to pick up units inside vehicles.

Reply:
The main idea here, is to check if player is around. 
Then something like this should fit the intended behavior:
`allPlayers inAreaArray [getPosWorld _unit, _distance/2, _distance/2]`

Reply:
`if ((_targets findIf {isPlayer _x}) == -1) exitWith {};`

Reply:
> There always is at least one unit near, the unit that is hurt.

This is about AI moaning if I understand correctly.

> Also, the avatar should still hear himself moan, even if there is no one else nearby.

AI can't hear itself.

Reply:
jonpas, this script is not for AI only, it is for everyone.

Reply:
I know, but the goal of the PR (if I understood correctly) is to disable it when an AI moans but no player is in range to hear it.

Reply:
And the line I commented on does not work that way. This argument is stupid.

Reply:
are you sure that nearEntities is cheaper, than just paying the 300us cost to let the function run?

Reply:
It is, I checked 
You can see in the screen the time of the `nearEntities` in purple
![Sans titre](https://user-images.githubusercontent.com/14364400/74338650-e594db00-4da2-11ea-8455-5ad35b8c7402.png)
If I remembrer it is around 50µs in this case.

Reply:
Perf of `nearEntities` is irrelevant. Unless you do shenigans with `crew`, the command is unusable for this, as I said already.

Reply:
isn't filtering out crew desirable?
people inside of a tank shouldn't hear the cries of enemies lubricating their treads

Reply:
> You can see in the screen the time of the nearEntities in purple

*squint* sooo teeeny